config.example.yml

```diff
@@ -17,3 +17,6 @@
     LOG_LEVEL: INFO
     LOG_FILE: bot.log
     LOG_TO_CONSOLE: True
+  TELEGRAM:
```

Comment:
Could you put some instruction here on where to find `id` and `token`? Telegram probably has some documentation and a link to that will probably suffice.

Reply:
ok, gonna add more details